From 972c34465d1f78cc2e2ebe53192905d378c1541e Mon Sep 17 00:00:00 2001
From: Roberto Oliveira <robertoguimaraes8@gmail.com>
Date: Sun, 29 Oct 2017 14:05:48 +0000
Subject: [PATCH] rbd-fuse: make sure PATH_MAX is defined

On systems without glibc, as Alpine Linux, you might lack definition of
PATH_MAX. This patch adds the limits.h header to solve this issue.

Signed-off-by: Roberto Oliveira <robertoguimaraes8@gmail.com>
--- a/src/rbd_fuse/rbd-fuse.cc
+++ b/src/rbd_fuse/rbd-fuse.cc
@@ -19,6 +19,7 @@
 #include <getopt.h>
 #include <assert.h>
 #include <string>
+#include <limits.h>
 
 #if defined(__FreeBSD__)
 #include <sys/param.h>
